docs/DeveloperGuide.md

```diff
@@ -256,47 +256,67 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
 Priority | As a ... | I want to ... | So that I can...
```

Comment:
@LiXiaoooowei You all did not update the diagrams? If you do please update the diagram here :)